datasketches-memory-java8/src/test/java/org/apache/datasketches/memory/internal/ExampleMemoryRequestServerTest.java

```diff
@@ -107,6 +113,6 @@ public void printlnTest() {
    * @param o value to print
    */
   static void println(Object o) {
-    //System.out.println(o); //disable here
+    System.out.println(o); //disable here
```

Comment:
this should be commented out by default and enabled manually when doing targeted testing

Reply:
Right. Fixed.